src/q2g-ext-listboxExtension.ts

```diff
@@ -0,0 +1,246 @@
+//#region Imports
+import "css!./q2g-ext-listboxExtension.css";
+
+import * as qvangular from "qvangular";
+import * as qlik from "qlik";
+import * as template from "text!./q2g-ext-listboxExtension.html";
+
+import { utils, logging, services, version } from "../node_modules/davinci.js/dist/umd/daVinci";
+
+import { RootExtension } from "../node_modules/davinci.js/src/utils/rootclasses";
+import { ListboxDirectiveFactory } from "./q2g-ext-listboxDirective";
+//#endregion
+
+//#region registrate services
+qvangular.service<services.IRegistrationProvider>("$registrationProvider", services.RegistrationProvider)
+.implementObject(qvangular);
+//#endregion
+
+//#region Logger
+logging.LogConfig.SetLogLevel("*", logging.LogLevel.info);
+let logger = new logging.Logger("Main");
+//#endregion
+
+//#region registrate directives
+var $injector = qvangular.$injector;
+utils.checkDirectiveIsRegistrated($injector, qvangular, "", ListboxDirectiveFactory("Listboxextension"),
+    "ListboxExtension");
+//#endregion
+
+//#region interfaces
+interface IDataProperties {
```

Comment:
@thomashaenig I see no advantage compared to a direct :any in the show function below, or you can add a :{ properties: any }